packages/runtime/src/interpreter/interpreter.ts

```diff
@@ -192,9 +198,10 @@ export class Interpreter {
 	 * @param accountRef index of account to fetch from account list
 	 * @param line line number
 	 * @param create create flag, default is true
+	 * @param imutable allow to access foreign application account, default is true
```

Comment:
its `immutable`. Could you provide more info why you chose this name?

Reply:
I choose this name because this name I pick up from the [go-algorand](https://github.com/algorand/go-algorand/pull/3994/files#diff-bd93d13a691c8de8507907073e8f2c62726a00ead3ee5095179c1c2b0901d51eR3128)

Reply:
I think those are just different test scenarios. IF you scroll below there is `Mutable` `Accessible` etc.

Reply:
those testcase is testing mutable and imutable references. I set the `imutable` variable into bool type so that it could prevent accessing foreign application from opcode `app_local_put`

Reply:
OK, so lets change it to `immutable` (its misspelled) and provide better comment for it so its clear what it does